src/commands/build/mod.rs

```diff
@@ -45,8 +45,13 @@ pub fn build(cache: &Cache, project_type: &ProjectType) -> Result<(), failure::E
                 failure::bail!("Webpack returned an error");
             }
 
+            let mut kv_namespaces = vec![];
+            if let Some(value) = &project.kv_namespaces {
+                kv_namespaces = value.clone();
+            }
```

Comment:
This whole block is more idiomatically written as:

    let kv_namespaces = project.kv_namespaces.unwrap_or(Vec::new());

(You may need a `.clone()` before the `.unwrap_or`, I'm not actually 100% sure)